devices/stm32u575.yaml

```
@@ -9,6 +9,60 @@ _modify:
   ADC12:
     name: ADC12_Common
 
+PWR:
+  CR2:
+    _remove: PKARAMPDS
+  _add:
+    CR4:
+      displayName: CR4
```

```suggestion
      displayName: PWR control register 4
```
Will apply all suggestions at the end.